build.gradle

```diff
@@ -73,3 +73,5 @@ subprojects {
 
 // Override spring boot's kotllin version dependency
 ext['kotlin.version'] = '${kotlinVersion}'
```

Comment:
Does the Kotlin version also need to be upgraded?

Reply:
> Does the Kotlin version also need to be upgraded?

No, and we already tested the build with Java 11 and current Kotlin version.